API/project.m

```diff
@@ -0,0 +1,43 @@
+function obj = project(options)
+    % Creates a project object.
+    % The input arguments are the experiment name which is a char
+    % array; the calculation type, which is a calculationTypes enum; and
+    % the geometry, which is a geometryOptions enum. All of the arguments
+    % are optional.
+    %
+    % problem = project(name='New experiment', calc='nonPolarised');
+    arguments
+        options.name {mustBeTextScalar} = ''
+        options.calc = calculationTypes.NonPolarised
```

Comment:
This is more or less the project type so `options.type` might be better then `options.calc`

Reply:
Yeah, I was considering both of those as well.